operator/src/main/java/oracle/kubernetes/operator/TuningParameters.java

```diff
@@ -258,12 +258,13 @@ class PodTuning {
     public final int livenessProbeFailureThreshold;
     public final long introspectorJobActiveDeadlineSeconds;
     public final long maxReadyWaitTimeSeconds;
+    public final boolean restartEvictedPods;
```

Comment:
Is the default false? It seems like the default behavior should be to restart evicted pods. I'll remove this comment if I find that it's initialized to "true" below.